future.tex

```diff
@@ -15,7 +15,6 @@ \chapter{Future Ideas}
       be halted/run/stepped simultaneously. When a hart hits a breakpoint, all
       the other harts in the same group also halt within a few clock cycles.
    \item DTMs are specified for protocols like USB, I2C, SPI, and SWD.
-   \item Core registers can be read without halting the processor.
    \item The debugger can communicate with the power manager to power cores up
       or down, and to query their status.
    \item Serial ports can raise an interrupt when a send/receive queue becomes full/empty.
```

Comment:
maybe we should delete this one since there is nothing about Serial Ports in the spec at all, while we're cleaning this up anyway